src/test/java/com/redhat/cloud/notifications/NotificationsAppTest.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.cloud.notifications;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NotificationsAppTest {
+
+    final String regex = ".*(/health|/metrics) HTTP/[0-9].[0-9]\\\" 200.*";
```

Comment:
The tests should use `NotificationsApp.REGEX_FILTER` rather than declaring a duplicate of the regex here.